src/index.html

```diff
@@ -1,12 +1,355 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
+    <title>Document</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a class="header__link" href="#home">
+          <img src="./images/logo.png" alt="logo company">
+        </a>
+      </div>
+
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a class="header__nav-link is-active"
+            href="#1">Apple
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a class="header__nav-link"
+            href="#2">Samsung
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a class="header__nav-link"
```

Comment:
Please check for all cases!
![image](https://user-images.githubusercontent.com/54936044/225773145-27cd10db-b02c-4a02-97c8-4e9320179568.png)